backend/apis/models.py

```diff
@@ -3,18 +3,21 @@
 from werkzeug.security import generate_password_hash, check_password_hash
 
 
+
+DB_STRING_LENGTH = 120
+
 class User(UserMixin, db.Model):
     id = db.Column(db.Integer, primary_key=True)
-    username = db.Column(db.String(80), unique=True, nullable=False)
```

Comment:
should we have another global variable for the 80?

Reply:
The initial number was arbitrary @ericong18 . It was a placeholder from the beginning.

Reply:
ok @adialachar